### ISSUE TITLE
SuperCam view looks incorrect

### ISSUE THREAD

**Issue by gkjohnson** [NASA-AMMOS/CameraModelUtilsJS#30]
<img width="785" alt="image" src="https://github.com/NASA-AMMOS/CameraModelUtilsJS/assets/734200/6daf078d-e6e3-4ba0-8a51-71bf5c08995d">

**Comment by nuernber** [comment]
The checkerboard rendering pattern maybe looks suspicious: 

<img width="453" alt="image" src="https://github.com/NASA-AMMOS/CameraModelUtilsJS/assets/276325/59b9f4ac-af84-454c-9fef-c1ae444b33c2">

**Comment by gkjohnson** [comment]
Yeah I saw that, as well. I'll have to take a closer look when I get the chance. It's possible the camera parameters have some kind of quirk to them.
